src/ixdat/calculators/ms_calculators.py

```diff
@@ -0,0 +1,829 @@
+            if axis_measurement:
+                if remove_bg_on_axis_measurement:
+                    t_plot, S_plot = t, S
+                else:
+                    t_plot, S_plot = measurement.grab_signal(mass, tspan=tspan)
+                axis_measurement.plot(
+                    t_plot, S_plot, color=STANDARD_COLORS[mass], linewidth=5
+                )
+            n_dot = (
+                inlet.calc_n_dot_0(gas=carrier_mol, p=pressure) * mol_conc_ppm / 10**6
+            )
+            S_list.append(np.mean(S))
+            n_dot_list.append(n_dot)
+        n_dot_vec = np.array(n_dot_list)
+        S_vec = np.array(S_list)
+        pfit = np.polyfit(n_dot_vec, S_vec, deg=1)
+        F = pfit[0]
+        if ax:
+            color = STANDARD_COLORS[mass]
+            if ax == "new":
+                ax = measurement.plotter.new_ax(
+                    xlabel="molecule flux / [nmol/s]", ylabel="signal / [nA]"
+                )
+            ax.plot(n_dot_vec * 1e9, S_vec * 1e9, "o", color=color)
+            n_dot_fit = np.array([0, max(n_dot_vec)])
+            S_fit = n_dot_fit * pfit[0] + pfit[1]
+            ax.plot(n_dot_fit * 1e9, S_fit * 1e9, "--", color=color)
+        cal_result = MSCalResult(
+            name=f"{mol}@{mass}",
+            mol=mol,
+            mass=mass,
+            cal_type=cal_type,
+            F=F,
+        )
+        return cls(ms_cal_results=[cal_result], measurement=measurement)
+
+    @property
+    def ms_cal_result_ids(self):
+        return [cal.id for cal in self.ms_cal_results]
+
+    @property
+    def mol_list(self):
+        return list({cal.mol for cal in self.ms_cal_results})
+
+    @property
+    def mass_list(self):
+        return list({cal.mass for cal in self.ms_cal_results})
+
+    @property
+    def name_list(self):
+        return list({cal.name for cal in self.ms_cal_results})
+
+    def __contains__(self, mol):
+        return mol in self.mol_list or mol in self.name_list
+
+    def __iter__(self):
+        yield from self.ms_cal_results
+
+    def __getattr__(self, attr):
+        """A MSCalibration with one cal result can be dropped in for that result:"""
+        if len(self.ms_cal_results) == 1:
+            ms_cal_result = self.ms_cal_results[0]
+            if hasattr(ms_cal_result, attr):
+                return getattr(ms_cal_result, attr)
+        raise AttributeError
+
+    @property
+    def available_series_names(self):
+        return set([f"n_dot_{mol}" for mol in self.mol_list])
+
+    def calculate_series(self, key, measurement=None):
+        """Return a calibrated series for `key` if possible.
+
+        If key starts with "n_", it is interpreted as a molecule flux. This method then
+        searches the calibration for a sensitivity factor for that molecule uses it to
+        divide the relevant mass signal from the measurement. Example acceptable keys:
+        "n_H2", "n_dot_H2".
```

Comment:
It does not differentiate between "n_H2" and "n_dot_H2" though does it? Be explicit about that in the docstring?

Reply:
maybe the first sentence is explicit enough.

Reply:
Docstring rewritten.